FreeRTOS/FreeRTOS
# Unit test fix for PR#625

Description
-----------
This PR adds fix for Unit Test for FreeRTOS+Kernel PR#[625](https://github.com/FreeRTOS/FreeRTOS-Kernel/pull/625) and duplicate PR#[719](https://github.com/FreeRTOS/FreeRTOS-Kernel/pull/719) 

Test Steps
-----------
```
make -C tasks
```

Checklist:
----------




By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.
